src/Renci.SshNet/Security/Cryptography/RsaCertificate.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using Renci.SshNet.Common;
+
+namespace Renci.SshNet.Security
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class RsaCertificate : IDisposable
```

Comment:
If there's no use in **CertificateHostAlgorithm** being public, we should also make this class internal.